### DISCUSSION TITLE
Center align latex equations in revealjs presentation

### DISCUSSION THREAD

**Discussion by sebastiantk** [quarto-dev/quarto-cli#8883] [Q&A] [answered]
### Description

Hello, 

Is there a way to center latex equations in revealjs presentations? 
At them moment they are left aligned. 

Thank you!

**Comment by mcanouil** [comment]
Could you share a small **self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Comment by sebastiantk** [comment]
```
---
title: "Example"
format: 
    revealjs:
        theme: default
        html-math-method: mathjax
---


## Equations

This is a centered equation:
\begin{align}
 a^2+b^2=c^2
\end{align}

* This is a bullet point:
  \begin{align}
   a^2+b^2=c^2
    \end{align}
---

## Equations 2

Adding longer text indents the equations more:
    
* This is a bullet point:
  \begin{align}
   a^2+b^2=c^2
    \end{align}
* Making the bullet point longer indents the equation more:
  \begin{align}
   a^2+b^2=c^2
    \end{align}
```

The issue appears with equations in lists, but also not consistently. On slide two the equation in the list is not indented enough, whereas on slide three they seem correctly aligned.

**Reply by mcanouil** [reply]
Could you remove all unnecessary options and comments? Also I don't have "custom".
Note that it's expected to have equation behaves that way when inside a list.
Also note that you should use `$$` to delimit equations.

**Reply by sebastiantk** [reply]
Apologies, I forgot about the custom file. It only change by how much the font size is reduced if smaller is enabled. 
I updated the example and the issue still persists.

Is there a way to overide this behavior? To me it would be preferable to have the equations center aligned even if they are in a list.

**Reply by mcanouil** [reply, marked as answer]
Short answer, you need to use CSS and target the right element using your favourite browser inspect/developer mode (I am no longer in front of a keyboard)

**Reply by sebastiantk** [reply]
I see. Thank you!
